test/gui/shared/steps/steps.py

```diff
@@ -67,6 +67,16 @@ def hook(context):
         pass
 
 
+@Given('the user has added an account with')
+def step(context):
+    clickButton(
+        waitForObject(names.settings_settingsdialog_toolbutton_Add_account_QToolButton)
```

Comment:
shouldn't that be part of the page object?

Reply:
Fixed